workflow-HSMetrics/src/main/filters/workflow.properties

```diff
@@ -0,0 +1,3 @@
+workflow-version=${project.version}
+workflow-description=This workflow runs vasrcan2 tool to generate SNPS, Indels and copy number calls. It uses samtools,java and picard tools.
```

Comment:
Can this description be updated for HSMetrics?

Reply:
fixed